src/MinterGateway.sol

```diff
@@ -779,8 +779,17 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
 
         if (principalOfMaxAllowedActiveOwedM_ >= principalOfActiveOwedM_) return;
 
+        if (updateCollateralInterval() == 0) return; // TODO: add note, fixes inconsistency for penalties
+
+        uint256 timeDelta_ = UIntMath.min32(
```

Comment:
In this case, the Minter would be penalized at most for one update collateral interval.
So as a Minter, I would potentially risk to be deactivated by governance but I could also wait several update intervals to fix my collateralization. I would still incur penalty for missed collateral updates but maybe it makes sense economically to wait.